plugin/client.go

```diff
@@ -31,11 +36,13 @@ func ClientMain(pluginImplementation interface{}) {
 
 type MattermostPlugin struct {
 	// API exposes the plugin api, and becomes available just prior to the OnActive hook.
-	API API
+	API     API
+	Helpers Helpers
```

Comment:
I can't think of a better term either.

But maybe use the singular `Helper` instead?

Reply:
Not sure that's any better.

Reply:
What was the advantage of making this a named field? `p.Helper.X` being more explicit than `p.X()`?

Reply:
> What was the advantage of making this a named field? `p.Helper.X` being more explicit than `p.X()`? 0/5?

Pretty much this. This way we make sure that devs don't mix up the helper methods with there own plugin methods.

Reply:
Exactly.